waspc/src/Generator/ExternalCodeGenerator/Js.hs

```diff
@@ -0,0 +1,16 @@
+module Generator.ExternalCodeGenerator.Js
+       ( generateJsFile
+       ) where
+
+import qualified Path
+import qualified Generator.FileDraft as FD
+import qualified ExternalCode as EC
+import qualified Generator.ExternalCodeGenerator.Common as C
+
+generateJsFile :: C.ExternalCodeGeneratorStrategy -> EC.File -> FD.FileDraft
+generateJsFile strategy file = FD.createTextFileDraft dstPath text'
+  where
+    text = EC.fileText file
```

Comment:
do you need this as an extra line, maybe just put inline below?

Reply:
Sure, done! I wanted to emphasize: this is `text`, and this is `text'`, but ok.

Reply:
Actually I like that "emphasis", I will put it back as it was :D.